src/quadrature/Galerkin_Ordinate_Space.cc

```diff
@@ -551,7 +553,8 @@ Galerkin_Ordinate_Space::compute_M_SN(vector<Ordinate> const &ordinates) {
                               << "   " << ordinates[m].xi()
                               << "   " << ordinates[m].wt()
                               << "   " << polar
-                              << "   " << azimuthal*180.0/3.141592653589793238462643383279
+                              << "   "
+                         << azimuthal*180.0/3.141592653589793238462643383279
                               << std::endl;
```

Comment:
I'm guessing clang-format did this wonky indentation? This looks like debug printing, so maybe it can go away altogether at some point.